pkg/cfg/config.go

```diff
@@ -74,6 +74,21 @@ func NewWithInterfaces(lookupEnv LookupEnv) *config {
 	return cfg
 }
 
+// load and merge multiple configuration files
+func FromFiles(filePaths []string, fileType string) GosoConf {
+	cfg := New()
+
+	for _, filePath := range filePaths {
+		err := WithConfigFile(filePath, fileType)(cfg.(*config))
```

Comment:
I did clean that up already in my branch. I would suggest to use the `Option` method instead. 
```suggestion
		err := cfg.Option(WithConfigFile(filePath, fileType))
```

Reply:
Actually, i have misread it.. thought the function would have a config receiver and would be in the interface..

So this one is not as bad as I thought.. :grimacing:

Nevertheless, I would move this one to the test package as this is more of an edge case or special use case?